capitalize_decorator.rb

```diff
@@ -0,0 +1,14 @@
+# capitalize_decorator.rb
+require_relative 'nameable'
+
+# Capitalize decorator
+class CapitalizeDecorator < Nameable
+  def initialize(nameable)
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Prepare%20base%20Decorator), Please create the `base decorator` class that:
  - Make sure that it inherits from Nameable.
  - In the constructor assign a nameable object from params to an instance variable.
  - Implement the correct_name method that returns the result of the correct_name method of the `@nameable.`

and then kindly make the `CapitalizeDecorator` and `TrimmerDecorator` classes inherit from the `BaseDecorator` class.